CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
 - **Dynamic Dashboards:** Expand dashboards_init_dashboard_completed tracking info [#111102](https://github.com/grafana/grafana/pull/111102), [@idastambuk](https://github.com/idastambuk)
 - **ErrorBoundary:** Report specific boundary type to Faro [#112071](https://github.com/grafana/grafana/pull/112071), [@tskarhed](https://github.com/tskarhed)
 - **Explore:** Use compact mode only when targeting Tempo [#113037](https://github.com/grafana/grafana/pull/113037), [@ifrost](https://github.com/ifrost)
+- **Explore:** Fix issue where tags were double escaped before passing to Tempo resulting in errors. [#114275](https://github.com/grafana/grafana/pull/114275), [@joe-elliott](https://github.com/joe-elliott)
```

Comment:
@joe-elliott the `add-to-changelog` label will add it to the changelog once it is released. It is going to use the title of the PR.